yarn.lock

```diff
@@ -156,6 +156,195 @@
     prettier "^1.19.1"
     yargs "^15.1.0"
 
+"@aws-amplify/graphql-function-transformer@0.4.3":
```

Comment:
This file will need to be updated once all of the dependencies are bumped to their latest versions.